<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdt</class>
 <widget class="QMainWindow" name="mdt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mdt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="3">
     <widget class="QLabel" name="label_Information">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Display Information</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_Next">
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
    min-width: 80px;
	font-size:15pt; font-weight:300

}

QPushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}


QPushButton:pressed {
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #fff);
}</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_Back">
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
    min-width: 80px;
	font-size:15pt; font-weight:300
}

QPushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}


QPushButton:pressed {
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #fff);
}</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>586</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLabel" name="label_Top">
      <property name="text">
       <string>Question:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QLabel" name="label_Image">
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_Start">
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
    min-width: 80px;
	font-size:15pt; font-weight:300
}

QPushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}

QPushButton:pressed {
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #fff);
}</string>
      </property>
      <property name="text">
       <string>Start!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
    min-width: 80px;
	font-size:15pt; font-weight:300
}

QPushButton:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}


QPushButton:pressed {
    background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #fff);
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QLabel" name="label_Process_Info">
      <property name="text">
       <string>Current-Process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMagic_device_ttol">
    <property name="title">
     <string>Magic-device-tool</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMagic_device_ttol"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_Next</tabstop>
  <tabstop>pushButton_Back</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>pushButton_Start</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
